test/Ocelot.UnitTests/Configuration/DefaultMetadataCreatorTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Ocelot.Configuration;
+using Ocelot.Configuration.Creator;
+using Ocelot.Configuration.File;
+
+namespace Ocelot.UnitTests.Configuration;
```

Comment:
I'm a bit confused here. On one hand, we are testing the extended configuration, but on the other, we are testing the Metadata feature. Well, why not move it to a separate `Metadata` folder?

Reply:
I don't get your point, it was the logic until now. I placed the creator test in the creator folder, that's it.